Description: Fix quoting in autoconf macro files
 Avoids warnings from autoreconf, which can break unrelated builds on
 the system when those builds require autoreconf's --warnings=error
 flag.
Author: Steve M. Robbins <steven.robbins@videotron.ca>
Origin: https://bugs.debian.org/302907#5
Forwarded: yes
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- libstroke-0.5.1.orig/libstroke.m4	2003-11-12 07:45:28.000000000 -0500
+++ libstroke-0.5.1/libstroke.m4	2005-04-03 14:35:16.709447679 -0400
@@ -26,7 +26,7 @@
 dnl                     if different from name
 dnl description         (optional) used to construct help string
 dnl
-AC_DEFUN(smr_ARG_WITHLIB, [
+AC_DEFUN([smr_ARG_WITHLIB], [
 
 ifelse($2, , smr_lib=[$1], smr_lib=[$2])
 
@@ -74,7 +74,7 @@
 dnl extra-flags         (optional) flags required when compiling the
 dnl                     header, typically more includes; for ex. X_CFLAGS
 dnl
-AC_DEFUN(smr_ARG_WITHINCLUDES, [
+AC_DEFUN([smr_ARG_WITHINCLUDES], [
 
 AC_ARG_WITH([$1]-includes,
 [  --with-$1-includes=DIR  set directory for $1 headers],
@@ -116,7 +116,7 @@
 dnl x-libs      (optional) extra libraries, if needed to link with lib
 dnl x-flags     (optional) extra flags, if needed to include header files
 dnl
-AC_DEFUN(smr_CHECK_LIB,
+AC_DEFUN([smr_CHECK_LIB],
 [
 ifelse($2, , smr_lib=[$1], smr_lib=[$2])
 ifelse($5, , , smr_header=[$5])
--- libstroke-0.5.1.orig/libgstroke.m4	2003-11-12 07:45:28.000000000 -0500
+++ libstroke-0.5.1/libgstroke.m4	2005-04-03 14:35:16.709447679 -0400
@@ -26,7 +26,7 @@
 dnl                     if different from name
 dnl description         (optional) used to construct help string
 dnl
-AC_DEFUN(smr_ARG_WITHLIB, [
+AC_DEFUN([smr_ARG_WITHLIB], [

 ifelse($2, , smr_lib=[$1], smr_lib=[$2])

@@ -74,7 +74,7 @@
 dnl extra-flags         (optional) flags required when compiling the
 dnl                     header, typically more includes; for ex. X_CFLAGS
 dnl
-AC_DEFUN(smr_ARG_WITHINCLUDES, [
+AC_DEFUN([smr_ARG_WITHINCLUDES], [

 AC_ARG_WITH([$1]-includes,
 [  --with-$1-includes=DIR  set directory for $1 headers],
@@ -116,7 +116,7 @@
 dnl x-libs      (optional) extra libraries, if needed to link with lib
 dnl x-flags     (optional) extra flags, if needed to include header files
 dnl
-AC_DEFUN(smr_CHECK_LIB,
+AC_DEFUN([smr_CHECK_LIB],
 [
 ifelse($2, , smr_lib=[$1], smr_lib=[$2])
 ifelse($5, , , smr_header=[$5])
